monitoring/mongodb/dashboard.py

```diff
@@ -277,31 +551,57 @@ def mongodb_state_timeline(title, expr, description="", mappings=None, **kwargs)
     unit=UNITS.BYTES,
     targets=[
         Target(
-            expr=f'sum(mongodb_dbstats_indexSize{{{DATABASE_FILTER_USER}, job=~"$job", rs_state="{RS_STATE_PRIMARY}", namespace="{NAMESPACE}"}})',
+            expr="sum("
+            + Metrics.DBSTATS_INDEX_SIZE(
+                DATABASE_FILTER_USER, f'rs_state="{RS_STATE_PRIMARY}"'
+            )
+            + ")",
             legendFormat="Indexes",
         ),
         Target(
-            expr=f'sum(mongodb_dbstats_fsUsedSize{{{DATABASE_FILTER_USER}, job=~"$job", rs_state="{RS_STATE_PRIMARY}", namespace="{NAMESPACE}"}} - mongodb_dbstats_indexSize{{{DATABASE_FILTER_USER}, job=~"$job", rs_state="{RS_STATE_PRIMARY}", namespace="{NAMESPACE}"}})',
+            expr="sum("
+            + Metrics.DBSTATS_FS_USED_SIZE(
+                DATABASE_FILTER_USER, f'rs_state="{RS_STATE_PRIMARY}"'
```

Comment:
as we set `rs_state` to `RS_STATE_PRIMARY` most of the type, maybe we should set it by default to this value (directly in metrics parameter definition, like `namespace`)